### ISSUE TITLE
bug: Setting the bodySmall and bodyLarge TextTheme paramaters does not affect the confirm button text color

### ISSUE THREAD

**Issue by vanishedvan** [LeGoffMael/insta_assets_picker#13]
When creating theme data with the following parameters:
```dart
textTheme: TextTheme(
    bodyLarge: TextStyle(color: Colors.white),
    bodySmall: TextStyle(color: Colors.white),
)
```
...the confirm button text style does not get affected.

**Comment by vanishedvan** [comment]
Decided to apply the fix myself, see #14. Should be fixed, hopefully.

**Comment by LeGoffMael** [comment]
Hello, thank you for trying this package.

I think using `textButtonTheme` might be better to edit this button, (we might not want to affect the other texts)
Would something like this works for you ?
```dart
pickerTheme: InstaAssetPicker.themeData(Theme.of(context).primaryColor).copyWith(
  // using `textButtonTheme` to edit the confirm button
  textButtonTheme: TextButtonThemeData(
    style: TextButton.styleFrom(
      foregroundColor: Colors.blue,
      disabledForegroundColor: Colors.red,
      textStyle: const TextStyle(
        color: Colors.blue,
        fontWeight: FontWeight.bold,
      ),
    ),
  ),
),
```

**Comment by vanishedvan** [comment]
Not sure why, but the `foregroundColor` parameter does not affect the Confirm text color when a picture is selected, stays blue no matter what. The `disabledForegroundColor` parameter works fine. Here's how I'm applying the picker theme below:

```dart
pickerTheme: theme.copyWith(
    canvasColor: Colors.black, // body background color
    splashColor: Colors.white.withOpacity(.25), // ontap splash color
    textButtonTheme: TextButtonThemeData(
        style: TextButton.styleFrom(
            foregroundColor: Colors.pink,
            disabledForegroundColor: Colors.amber,
        ),
    ),
    colorScheme: theme.colorScheme.copyWith(
        background: Colors.black, // albums list background color
    ),
    appBarTheme: theme.appBarTheme.copyWith(
        backgroundColor: Colors.black, // app bar background color
        titleTextStyle:
            Theme.of(context).appBarTheme.titleTextStyle?.copyWith(
                color: Colors.white,
            ), // change app bar title text style to be like app theme
    ),
),
```
(I'm using pink and amber for testing purposes)

**Comment by LeGoffMael** [comment]
Did the version you use included the change i [mentioned](https://github.com/LeGoffMael/insta_assets_picker/commit/0d1f37afdae988c055e8349fad44fd7d7ada1f7a) ?

```yaml
insta_assets_picker:
  git:
    url: https://github.com/LeGoffMael/insta_assets_picker
    ref: fix/confirm-btn-textButtonTheme
```

**Comment by vanishedvan** [comment]
Yep! Did manage to fix it later though, had an unrelated bug. Everything seems to be working, I appreciate it! You can push the new version now

**Comment by LeGoffMael** [comment]
Change published in the new 1.5.1 version
